Frontend/src/Components/FileExplorerContainer/FileExplorerContainer.jsx

```diff
@@ -50,47 +68,49 @@ const FileExplorerContainer = () => {
     if (currentFolder && currentFolder.parentFolder) {
       fetchFolderContents(currentFolder.parentFolder);
     } else {
-      fetchFolderContents('root');
+      fetchFolderContents("root");
     }
   };
 
   const handleSearch = (event) => {
     setSearchTerm(event.target.value);
   };
 
-  const filteredContents = contents.filter(item => 
+  const filteredContents = contents.filter((item) =>
     item.name.toLowerCase().includes(searchTerm.toLowerCase())
   );
 
   const handleCreateFolder = async () => {
-    const folderName = prompt('Enter folder name:');
+    const folderName = prompt("Enter folder name:");
     if (!folderName) return;
 
     try {
-      const token = localStorage.getItem('auth-token');
       const response = await axios.post(
-        'http://localhost:8001/api/user/Teacher/file-folder/createFolder',
+        "http://localhost:8001/api/user/Teacher/file-folder/createFolder",
         {
           name: folderName,
           parentFolderId: currentFolder._id,
         },
         {
           headers: {
-            'Authorization': `Bearer ${token}`,
-            'Content-Type': 'application/json',
+            Authorization: `Bearer ${token}`,

```

Comment:
Just reccomending a slight changes, put the `authorization` in single quotes as before.